docker/letsencrypt/crontab

```diff
@@ -1,9 +1,8 @@
-# .............. minute (0 - 59)
-# .     .............. hour (0 - 23)
-# .     .     .............. day of month (1 - 31)
-# .     .     .     .............. month (1 - 12)
-# .     .     .     .     .............. day of week (0 - 6) (Sunday to Saturday; 7 is also Sunday on some systems)
-# .     .     .     .     .
+# +------------- minute (0 - 59)
+# �     +------------- hour (0 - 23)
+# �     �     +------------- day of month (1 - 31)
```

Comment:
Is the � an error due to the GitHub view or is it a real unwanted character in the code?